hazelcast/src/test/resources/log4j.properties

```diff
@@ -28,6 +28,11 @@ log4j.logger.com.hazelcast.spi.hotrestart=info
 log4j.logger.com.hazelcast.spi.hotrestart.cluster=debug
 log4j.logger.com.hazelcast.test.mocknetwork=debug
 
+#log4j.logger.com.hazelcast.internal.nearcache.impl.invalidation=trace
```

Comment:
leftover?

Reply:
I have intentionally put it here. I have merged client and core log4j properties. These are probably left here to open related logs easily.